upup/pkg/fi/cloudup/gcetasks/instancetemplate.go

```diff
@@ -238,6 +240,14 @@ func (_ *InstanceTemplate) CheckChanges(a, e, changes *InstanceTemplate) error {
 	if fi.StringValue(e.MachineType) == "" {
 		return fi.RequiredField("MachineType")
 	}
+	if e.ProvisioningModel != nil {
+		if fi.StringValue(e.ProvisioningModel) != "SPOT" && fi.StringValue(e.ProvisioningModel) != "STANDARD" {
+			return fmt.Errorf("provisioningModel must be SPOT or STANDARD")
+		}
```

Comment:
This should be checked during validation.

Reply:
fixed